tests/integ/test_airflow_config.py

```diff
@@ -614,7 +614,8 @@ def test_pytorch_airflow_config_uploads_data_source_to_s3_when_inputs_not_provid
         estimator = PyTorch(
             entry_point=PYTORCH_MNIST_SCRIPT,
             role=ROLE,
-            framework_version="1.1.0",
+            framework_version="1.3.1",
```

Comment:
Just curious, why is here "1.3.1" instead of the latest training version?

Reply:
will update